frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -1,7 +1,8 @@
 import { ExplorerApi } from ".";
 
-interface AccountId {
+export interface AccountId {
```

Comment:
It seems to make sense to rename this to just `Account`.

Reply:
but we already name this `export type Account = AccountId & AccountStats & AccountInfo;` is it good to name interface and type same?

Reply:
Hmm, my bad. Well, we should come up with a better name here as it does not return only AccountId anymore, and it was a bad name in the first place. The only name I, currently, have in mind is `AccountBasicInfo`.